module/zfs/zfs_ioctl.c

```diff
@@ -7399,7 +7470,8 @@ zfsdev_ioctl_common(uint_t vecnum, zfs_cmd_t *zc, int flag)
 		return (SET_ERROR(ZFS_ERR_IOC_CMD_UNAVAIL));
 
 	zc->zc_iflags = flag & FKIOCTL;
-	if (zc->zc_nvlist_src_size > MAX_NVLIST_SRC_SIZE) {
```

Comment:
`MAX_NVLIST_SRC_SIZE` is now unused and the define can be removed.

Reply:
Yes it was removed.

Reply:
I might be able to remove the FreeBSD definition of `KMALLOC_MAX_SIZE` too, though. I'll test that out.